src/framework/uicomponents/qml/Muse/UiComponents/fuzzysortfilterproxymodel.cpp

```
@@ -0,0 +1,189 @@
+    connect(this, &QSortFilterProxyModel::filterCaseSensitivityChanged, this, &FuzzySortFilterProxyModel::compilePattern);
+    connect(this, &QAbstractItemModel::modelAboutToBeReset, this, &FuzzySortFilterProxyModel::clearScoreCache);
+    connect(this, &QSortFilterProxyModel::filterRoleChanged, this, &FuzzySortFilterProxyModel::clearScoreCache);
+    connect(this, &QAbstractProxyModel::dataChanged, this, &FuzzySortFilterProxyModel::clearScoreCache);
+}
+
+QString FuzzySortFilterProxyModel::fuzzyPattern() const
+{
+    return m_fuzzyPattern;
+}
+
+void FuzzySortFilterProxyModel::setFuzzyPattern(const QString fuzzyPattern)
```

Cppcheck (2.20.0)

[style] The function 'setFuzzyPattern' is never used.

(unusedFunction)